docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+Property Rights (IPR) Policy](https://cabforum.org/ipr-policy/), as such
+documents may change from time to time. This charter for the S/MIME Certificate
+Working Group has been created according to CAB Forum Bylaw 5.3.1. In the event
+of a conflict between this Charter and any provision in either the Bylaws or
+the IPR Policy, the provision in the Bylaws or IPR Policy SHALL take
+precedence. The definitions found in the Forum’s Bylaws SHALL apply to
+capitalized terms in this Charter.
+
+### 1. Scope
+
+The authorized scope of the SMCWG SHALL be to discuss, adopt, and maintain
+policies, frameworks, and sets of standards related to the issuance and
+management of S/MIME certificates by third-party CAs under a publicly trusted
+root, limited as follows:
+
+- Baseline verification of control over email addresses, including those used
+  by a natural person or a legal entity, or used by automated systems such as
+  for mailing lists
+- Identity validation for natural persons and legal entities
+- Key management and certificate lifecycle (subject to coordination with other
+  Forum CWGs to ensure consistency and avoid redundancy)
+- Certificate profiles for S/MIME certificates and Issuing CA certificates
+  (including the appropriateness of extensions and when those extensions should
+  be present)
+- CA operational practices, physical/logical security, etc.
+
+The SMCWG SHALL exercise caution to ensure that its work product does not impede
+the issuance of certificates with other EKU types.
+
+### 2. Out of Scope
+
+Certificates issued under a root certificate that is not publicly trusted SHALL
+be out of scope.
+
+### 3. Charter Expiration
+
+The SMCWG is chartered indefinitely until it is dissolved as specified in Bylaw
+5.3.2(c).
+
+### 4. Personnel and Participation
+
+#### 4.1. Selection of Officers
+
+Stephen Davidson will act as chair of the SMCWG until the first Working Group
+Teleconference, at which time the group will elect a chair and vice-chair. The
+chair and vice-chair will serve until October 31, 2022, or until they are
+replaced, resign, or are otherwise disqualified. Thereafter, elections SHALL be
+held for chair and vice chair every two years in coordination with the Forum’s
+election process and in conjunction with its election cycle. Voting SHALL occur
+in accordance with Bylaw 4.1(c).
+
+#### 4.2. Participation
+
+##### 4.2.1. Eligibility to Participate
+
+The SMCWG SHALL consist of two classes of voting members, Certificate Issuers
+and Certificate Consumers meeting the eligibility criteria below.
+
+1. A Certificate Issuer eligible for voting membership in the SMCWG MUST have
+   a publicly-available audit report or attestation statement in accordance
+   with one of the following schemes:
+
+    * WebTrust for CAs v.2.0 or newer; or
+    * ETSI EN 319 411-1, which includes normative references to ETSI EN 319 401
+      (the latest version of the referenced ETSI documents should be applied).
+
```

Comment:
As noted in https://cabforum.org/pipermail/public/2020-March/014917.html , this precludes a number of existing deployed S/MIME frameworks, including inter-organization frameworks (such as the US Federal PKI or SAFE-BioPharma)

While direct integration with these other PKIs is not necessarily a goal, it's not necessarily worth precluding a priori either.

Reply:
This is a good point, and one I had missed the context on. I would be reticent to support those frameworks once guidelines have been produced (transition periods accommodating, of course), but I agree there's not a need to preclude their participation in the CWG. With that understanding - assuming I have it right :| - in mind, I'd support adding that bullet back in.

Reply:
Yeah, this is really fluid depending on how the existing Certificate Consumers approach this. At the time this chartering was started, Certificate Consumers were still permissive here. If the goal is to produce something useful to the Certificate Consumers, and the Certificate Consumers do not already accept such audits / require different audits, then I suspect this isn't as valuable, since ultimately, this is about finding Guidelines that Certificate Consumers find broadly useful and acceptable for their needs.